app/models/lead_provider_delivery_partnership.rb

```diff
@@ -8,6 +8,7 @@ class LeadProviderDeliveryPartnership < ApplicationRecord
   has_one :lead_provider, through: :active_lead_provider
 
   touch -> { delivery_partner }, on_event: %i[create destroy], timestamp_attribute: :api_updated_at
+  touch -> { school_partnerships }, when_changing: %i[delivery_partner_id], timestamp_attribute: :api_updated_at
```

Comment:
Would this ever change? 🤔

Reply:
actually my bad, this was on the ticket, the change would be on `lead_provider_delivery_partnership_id` on school partnership intead rather than here

Reply:
Moved to `SchoolPartnership.lead_provider_delivery_partnership_id`